pkg/config/config.go

```diff
@@ -108,7 +112,8 @@ func UpdateConfigFilePath(configFile string) {
 	configFilePath = configFile
 }
 
-func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
+func LoadConfigFromFile() (*NSXOperatorConfig, error) {
+	log.Infof("loading NSX Operator configuration file: %s", configFilePath)
```

Comment:
why log.Infof? not log.Info?

Reply:
zap.SugaredLogger , the usage is different.